src/test/java/uk/gov/laa/pfla/ServiceManager.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.laa.pfla;
+
+import java.io.File;
+import java.io.IOException;
+
+public class ServiceManager {
```

Comment:
**Suggestion**: Add logging when starting and stopping the service.
**Reason**:
This provides visibility into the lifecycle of the service, which can be invaluable for debugging if the service fails to start or stop correctly.